source/position/position.go

```diff
@@ -0,0 +1,79 @@
+// Copyright © 2023 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package position
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+// IteratorType describe position type.
+type IteratorType string
+
+const (
+	TypeSnapshot = "s"
+	TypeCDC      = "c"
+)
+
+// Position represents SAP Hana position.
+type Position struct {
+	// IteratorType - shows in what iterator was created position.
+	IteratorType IteratorType
+
+	// Snapshot information.
+	// SnapshotLastProcessedVal - last processed value from ordering column.
+	SnapshotLastProcessedVal any
+	// SnapshotMaxValue - max value from ordering column.
+	SnapshotMaxValue any
+
+	// CDC information.
+	// CDCLastID - last processed id from tracking table.
+	CDCLastID int
+	// SuffixName special suffix that connector uses for identify tracking table and triggers.
+	SuffixName string
```

Comment:
Maybe we can use the full tracking table name?

Reply:
I was thinking about , but SuffixName is shorter  than tracking table name. For example for table name `SOME_REALLY _LONG_TABLE_NAME` tracking table name will be `COUNDUIT_SOME_REALLY _LONG_TABLE_NAME_203907` and suffix will be `203907`

Reply:
it is not big deal, I am going to change it